force_wfmanager/left_side_pane/tests/test_evaluator_model_view.py

```diff
@@ -61,7 +87,25 @@ def setUp(self):
         self.model = factory.create_model()
         self.evaluator = factory.create_kpi_calculator()
 
-        self.evaluator_mv = EvaluatorModelView(model=self.model)
+        self.variable_names_registry = VariableNamesRegistry()
+        self.variable_names_registry.data_source_available_variables = \
+            ['P1']
+        self.variable_names_registry.kpi_calculator_available_variables = \
+            ['P1', 'P2', 'P3']
+
+        self.evaluator_mv = EvaluatorModelView(
```

Comment:
Isn't the evaluatormodelview the base class of the two? Why are you using the base class, instead of the specific ones?